coalib/bearlib/aspects/Metadata.py

```diff
@@ -0,0 +1,217 @@
+from coalib.bearlib.aspects import Root, AspectDocumentation, AspectSetting
+
+
+Root.new_subaspect(
+    "Metadata",
+    AspectDocumentation(
+        definition="""
+        This describes any aspect that is related to metadata that is not
+        inside your source code.
+        """
+    )
+)
+
+Root.Metadata.new_subaspect(
+    "CommitMessage",
+    AspectDocumentation(
+        definition="""
+        Your commit message is important documentation associated with your
+        source code. It can help you to identify bugs (e.g. through
+        `git bisect`) or find missing information about unknown source code
+        (through `git blame`).
+
+        Commit messages are also sometimes used to generate - or write
+        manually - release notes.
+        """
+    )
+)
+
+Root.Metadata.CommitMessage.new_subaspect(
+    "Shortlog",
+    AspectDocumentation(
+        definition="""
+        Your commit shortlog is the first line of your commit message. It is
+        the most crucial part of your commit message and summarizes the change
+        in the shortest possible manner.
+        """
+    )
+)
+
+Root.Metadata.CommitMessage.Shortlog.new_subaspect(
+    "ColonExistence",
+    AspectDocumentation(
+        definition="""
+        Some projects force to use - or not use - colons in the commit message
+        shortlog (first line).
+        """,
+        cause="""
+        People often forget to obey the project guidelines for colons because
+        they don't know about them.
+        """,
+        example="""
+        FIX: Describe change further
+        context: Describe change further
+        """,
+        example_language="English",
+        importance_reason="""
+        The colon can be a useful separator for a context (e.g. a filename) so
+        the commit message makes more sense to the reader or a classification
+        (e.g. FIX, ...) or others. Some projects prefer not using colons
+        specifically: consistency is key.
+        """,
+        fix_suggestions="""
+        Add or remove the colon according to the commit message guidelines.
+        """
+    ),
+    settings=(AspectSetting(
+        "shortlog_colon",
+        "Wether or not the shortlog has to contain a colon.",
+        bool,
+        (True, False),
+        True
+    ),)
+)
+
+Root.Metadata.CommitMessage.Shortlog.new_subaspect(
+    "TrailingPeriod",
+    AspectDocumentation(
+        definition="""
+        Some projects force not to use - or use - trailing periods in the commit
```

Comment:
this sounds a bit cryptic - phrase it `Some projects force to use - or not use - ...`? I know that the default value is `False`, but this makes more sense IMO. We shouldn't need to change the description based on the default value.